impl/src/main/java/com/okta/sdk/impl/client/DefaultClientBuilder.java

```diff
@@ -210,6 +223,23 @@ else if (SYSPROPS_TOKEN.equalsIgnoreCase(location)) {
             clientConfig.setProxyPassword(props.get(DEFAULT_CLIENT_PROXY_PASSWORD_PROPERTY_NAME));
         }
 
+        if (Strings.hasText(props.get(DEFAULT_CLIENT_AUTHORIZATION_MODE_PROPERTY_NAME))) {
+            clientConfig.setAuthorizationMode(AuthorizationMode.getAuthorizationMode(props.get(DEFAULT_CLIENT_AUTHORIZATION_MODE_PROPERTY_NAME)));
+        }
+
+        if (Strings.hasText(props.get(DEFAULT_CLIENT_ID_PROPERTY_NAME))) {
+            clientConfig.setClientId(props.get(DEFAULT_CLIENT_ID_PROPERTY_NAME));
```

Comment:
We have a `ConfigurationValidator` method to validate a clientId, I'm not sure if the instructions are valid for this usage of a `clientId`
https://github.com/okta/okta-commons-java/blob/master/config-check/src/main/resources/com/okta/commons/configcheck/ConfigurationValidator.properties#L32-L33

cc: @laura-rodriguez

Reply:
nm, i see you have used it below :D